CMakeLists.txt

```diff
@@ -565,6 +565,12 @@ if(WITH_MYFILE_MMAP)
     add_definitions(-DMYFILE_MMAP)
 endif()
 
+# Atomic is required in some builds: #6821
+find_package(ATOMIC)
+if(ATOMIC_FOUND)
+	target_link_libraries(executable ${ATOMIC_LIBRARIES})
```

Comment:
@Benitoite `executable` wasn't meant literally. It should rather be `rawtherapee` and another line with `rawtherapee-cli` or completely different by adding `${ATOMIC_LIBRARIES}` to `EXTRA_LIB_RTGUI` and the corresponding variable for RT.

I'm surprised this works as is...

Reply:
#6832 has the properly named targets.